bin/steps/python

```diff
@@ -12,17 +12,34 @@ if [[ $PYTHON_VERSION =~ ^python-2 ]]; then
     puts-warn "The latest version of Python 2 is $LATEST_2 (you are using $PYTHON_VERSION, which is unsupported)."
     puts-warn "We recommend upgrading by specifying the latest version ($LATEST_2)."
     echo "       Learn More: https://devcenter.heroku.com/articles/python-runtimes"
+  else
+    echo "       Using supported version of Python 2 ($PYTHON_VERSION)"
```

Comment:
I believe this block is just for displaying warnings. Did you want to have this in addition to the rest of the buildpack output?

Reply:
I'd like to move the output in a direction where it's explicit (rather than silent) when a supported Python version is used. Open to chatting about other ways to go about that though - could be simplified to an operator for example